lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerLaunchTests.cs

```diff
@@ -25,6 +25,30 @@ public class PuppeteerLaunchTests : PuppeteerBaseTest
             {
                 var response = await page.GoToAsync(TestConstants.HttpsPrefix + "/empty.html");
                 Assert.Equal(HttpStatusCode.OK, response.Status);
+                Assert.NotNull(response.SecurityDetails);
+                Assert.Equal("TLS 1.2", response.SecurityDetails.Protocol);
+            }
+        }
+
+        [Fact]
+        public async Task NetworkRedirectsShouldReportSecurityDetails()
```

Comment:
maybe move this into a new class since it doesn't use the already launched browser

Reply:
Most of the tests on this class Launch a new browser.

Reply:
oops, I got this class mixed up with the page based classes...